dashboard/config/locales/en.yml

```diff
@@ -199,6 +199,7 @@ en:
     pair_programming: 'I have a partner at my computer'
     student_privacy: "Learn more about why you're not seeing your full name <a href='%{student_privacy_blog}' target='_blank'>here</a>."
   password:
+    invalid_password_entered: 'The password you entered is invalid.'
```

Comment:
This must already exist somewhere.  Can we find whatever existing 'bad password' message we have and reuse that?

Reply:
On login, we use a "Invalid email or password" message, which I think would be the closest existing string in dashboard. I could add the error to `current_user` and return that?
```
if password_required && invalid_password
  current_user.errors.add :current_password
  render json: {
    errors: current_user.errors.as_json(full_messages: true)
  }, status: :bad_request
  return
end
```

Reply:
I might prefer that just to be uniform and reuse existing translations.  I'm okay if you feel that's more complex than necessary though.

Reply:
no, i don't think that's too complex. it might actually make processing the error easier on the front-end. i'll refactor to that